.github/.workflows/fork-sync.yaml

```diff
@@ -0,0 +1,19 @@
+# This workflow will open a PR every 30 minutes to sync our develop branch with optimism's develop branch.
```

Comment:
When you say "open a PR", do you actually mean "open and merge a PR"? Presumably this is all automatic?

Reply:
it prob should be automatic yes. Right now it's not though (missing a PAT to allow the action to merge automatically). Will try this manual merging for now to see how it works, and then add the PAT later if everything's fine and merging by hand becomes annoying.